README.md

```diff
@@ -1,86 +1,149 @@
 # AMD MIGraphX
 
-AMD's graph optimization engine.
+AMD MIGraphX is AMD's graph inference engine to accelerate model inference on AMD GPUs. AMD MIGraphX can be used by
+installing binaries directly or building from source code.
+
+Note that all the following instructions are based on that ROCm has been installed successfully. ROCm installation
+instructions are explained in the [ROCm installation
+guide](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html).
+
+## Installing from binaries
+With ROCm installed correctly, MIGraphX binaries can be installed with the following command:
+```
+sudo apt update && sudo apt install -y migraphx
+```
+then the head files and libs are located at ```/opt/rocm/include``` and ```/opt/rocm/lib```, respectively, which can be
+included and linked by adding the corresponding folders to the Makefile.
+
+## Building from source
+
+Building MIGraphX from sources needs dependencies, which must be installed before building the source code. In the
+following, we first list the dependencies, and then explain two ways of installing them.
+
+### List of dependencies
+The following is a list of dependencies required to build MIGraphX from source code. This list is also available in the
+requirement files ```dev-requirements.txt``` and ```requirements.txt```.
 
-## Prerequisites
 * [ROCm cmake modules](https://github.com/RadeonOpenCompute/rocm-cmake) **required**
 * [MIOpen](https://github.com/ROCmSoftwarePlatform/MIOpen) for running on the GPU
+* [rocBLAS](https://github.com/ROCmSoftwarePlatform/rocBLAS) for running on the GPU
 * [HIP](https://github.com/ROCm-Developer-Tools/HIP) for running on the GPU
 * [Protobuf](https://github.com/google/protobuf) for reading [onnx](https://github.com/onnx/onnx) files
 * [Half](http://half.sourceforge.net/) - IEEE 754-based half-precision floating point library
 * [pybind11](https://pybind11.readthedocs.io/en/stable/) - for python bindings
+* [ONNX 1.7.0](https://github.com/onnx/onnx) and [Pytest](https://github.com/pytest-dev/pytest) for running the ONN backend
```

Comment:
This dependency should not be listed here, as this is only needed to run the tests, so most users will not need this.

Reply:
OK, I move it to another place.